src/math/Kconfig

```diff
@@ -9,6 +9,16 @@ config CORDIC_FIXED
 	  Select this to enable sin(), cos(), asin(), acos(),
 	  and cexp() functions as 16 bit and 32 bit versions.
 
+config MATH_LUT_TRIG_FIXED
+	bool "Lookup table based trigonometric functions"
+	default n
```

Comment:
doe it have dependency? I mean there should have one module use this, and this config should depend on that module.

Reply:
DRC component has "select MATH_LUT_TRIG_FIXED". OK? I will rename that as Sriram suggested but keep it that way.

Reply:
not sure need add: depends on COMP_DRC, because this table is large, if wrongly included by other modules,  it will occupy more space.

Reply:
I don't see how other modules would select this accidentally. E.g. MFCC and FFT use trig functions only in initialization code, where the cordic version as 32 bits is ideal.

If some other component would need sine function in hot code then this function should be available for them also. It's not making sense to make this private for DRC.